Formula/libngspice.rb

```diff
@@ -0,0 +1,79 @@
+class Libngspice < Formula
+  desc "Spice circuit simulator as shared library"
+  homepage "http://ngspice.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/ngspice/ng-spice-rework/26/ngspice-26.tar.gz"
+  sha256 "51e230c8b720802d93747bc580c0a29d1fb530f3dd06f213b6a700ca9a4d0108"
+
+  head do
+    url "git://git.code.sf.net/p/ngspice/ngspice"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "bison" => :build
+    depends_on "flex" => :build
+    depends_on "libtool" => :build
+
+    # Currently, headers don't get installed to include/*.
+    # There is a patch upstream that addresses this for HEAD.
+    # Upstream ticket: https://sourceforge.net/p/ngspice/bugs/327
+    patch :DATA
```

Comment:
That was what I tried at first, before trying the patch and the symlinking. I think the commenter upstream meant to say that --includedir=/foo does allow relocating the headers _after_ the patch has been applied.